lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -57,6 +57,21 @@ def stub_email_alert_api_has_subscription(id, frequency, optional_overrides = {}
           .to_return(status: 200, body: response)
       end
 
+      # Stubs the API responses as if each subscription happened in the order they are passed
+      def stub_email_alert_api_has_subscriptions(*subscriptions)
```

Comment:
What is the benefit of using the a splat here?

Reply:
Good point! None :)

Reply:
Because we pass subscriptions as a 0..* list, not an array

```ruby
stub_email_alert_api_has_subscriptions(
  [1, 'weekly', :ended]
  [2, 'daily']
)
```

Happy to change the API to take an array instead, i.e.

```ruby
stub_email_alert_api_has_subscriptions([
  [1, 'weekly', :ended]
  [2, 'daily']
])
```